### DISCUSSION TITLE
Increase endworm's minimal damage for severance to 550

### DISCUSSION THREAD

**Discussion by LePiromano** [FakeFishGames/Barotrauma#6750] [Ideas and suggestions]
Increasing the minimal damage for severance each limb has from 100 to 550 would make it so you the chance of segmenting the endworm with handheld weapons/low damage crewed weapons is little to zero.

why 550?
an empty railgun shell does 750 total damage (250 impact 500 explosion), altough the explosion usually fails to work and only deals half of its damage (now we have 500 total damage), meaning you wouldn't be able to segment endworms with empty common shells like you currently can (unless explosions decide to work correctly). This would make players more incentivized to use nuclear/physicorium shells or load explosives into the railgun shells.

**Comment by SpazzMaticus107** [comment]
I was really suprised how quickly you can kill the end worm. I wish it was a really strong mythical creature that would just fall back after losing strenght when fighting the submarine.
